tests/tst_degrib2_ftp_all.c

```diff
@@ -81,6 +81,68 @@ main()
         }
     }
     printf("ok!\n");
+#ifdef LARGE_FTP_TESTS
```

Comment:
Any reason to not consolidate this with the existing code? i.e., add to the file list if LARGE_FTP_TESTS is set?

Reply:
Just code simplicity. The large file FTP tests will rarely be run...